clang-tools-extra/docs/clang-tidy/GlobalOptions.rst

```diff
@@ -0,0 +1,21 @@
+==============
+Global Options
+==============
+
+Some options apply to multiple checks. This page lists all the available
+globally options.
+
+.. option:: IncludeStyle
+
+   A string specifying which include-style is used, `llvm` or `google`. Default
+   is `llvm`.
+
+.. option:: IgnoreMacros
```

Comment:
I think this one should  _not_ be global. The same is done for clang warnings. Sometimes it may make sense to ignore macros, sometimes not, depending on the check.

Reply:
according to current implement of clangtidy, lots of checks will read ignores macros from global scope. i don't think it is a good idea to change lots of checks behavior.

maybe i can notices something in this file to explain global option doesn't apply from every check?